### PR TITLE
Add allure bundle

### PR DESCRIPTION
**Pull request by eroshenkoam**

### CODE REVIEW

File: allure-bundle/pom.xml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            <type>war</type>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
```
Comment (baev) [inline allure-bundle/pom.xml]: add plugin version

File: allure-bundle/pom.xml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                            <outputDirectory>${allure.bundle.directory}/face/</outputDirectory>
+                            <includeArtifactIds>allure-report-face</includeArtifactIds>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <configuration>
```
Comment (baev) [inline allure-bundle/pom.xml]: add plugin version

**Comment by lanwen** [comment]
What this PR does?

**Comment by baev** [comment]
@lanwen adds new module. We are going to use allure-bundle.zip instead aether's resolving of `allure-report-data` and `allure-report-face`. 

As example we are going to add the following commands to  `allure-cli`:

`update` - load list of available versions of allure-report
`upgrade` - download latest `allure-bundle.zip` and cache it in `~/.allure folder`

And after it you be able to generate report without specifying report version and without internet connection.

In allure jenkins plugin we are going to implement interface like this:
 (list of available versions, ability to specify path to bundle and ability to auto download latest)

**Comment by lanwen** [comment]
:+1:

**Comment by baev** [comment]
Thanks!

**Comment by volkovs** [comment]
@baev does it mean that reports could be generated with Maven < 3.1.1 ?

**Comment by baev** [comment]
@volkovs I guess yep

**Comment by volkovs** [comment]
That would be really cool :+1: 

(lots of our clients are stuck with maven 3.0.5)

**Pull request merged**